README.md

```diff
@@ -1,2 +0,0 @@
-# android-peer-discovery
-Experiments with P2P on Android
```

Comment:
Projects should always have a README.md so that folks who run into it on GitHub have some idea what it is. Admittedly the currently description wasn't terribly good but at least it was a start.